Gemfile

```diff
@@ -3,6 +3,7 @@ source "https://rubygems.org"
 # Specify your gem's dependencies in x.gemspec
 gemspec
 
+gem "base64", ">=0.2"
```

Comment:
This is a nitpick but can you please add a space between the `>=` and the `0.2` and squash that into the same commit?

Reply:
Done!